openedx/features/edly/utils.py

```diff
@@ -333,3 +334,45 @@ def create_learner_link_with_permission_groups(user):
         user.groups.add(new_group)
 
     return user
+
+
+def get_current_site_invalid_certificate_context_data(default_html_certificate_configuration):
```

Comment:
Context and data are almost same thing. Please change the name to:
`get_current_site_invalid_certificate_context`

Reply:
Done